frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -115,6 +116,7 @@ export default class AccordionList extends Component {
     renderSectionExtra: () => null,
 
     // item getters/render props
+    itemExtraIsRightAligned: true,
```

Comment:
`itemExtraIsRightAligned` defaults to `true`, because in most cases the "item extra" (the component returned by `renderItemExtra`) is right aligned.